themes/antergos

```diff
@@ -1 +1 @@
-Subproject commit 1ffa21323cd362be664e09609da63aa3cf87ced2
+Subproject commit facac68f8cb59ed96d7e7ef0f6f9e4b3aafe8ca3
```

Comment:
Did you mean to do this are did you simply forget to do a `git submodule update` ?